druid-shell/src/platform/gtk/clipboard.rs

```diff
@@ -62,7 +62,25 @@ impl Clipboard {
     pub fn get_string(&self) -> Option<String> {
         let display = gdk::Display::get_default().unwrap();
         let clipboard = gtk::Clipboard::get_default(&display).unwrap();
-        clipboard.wait_for_text().map(|s| s.to_string())
+
+        let targets = [
+            "UTF8_STRING",
+            "TEXT",
+            "STRING",
+            "text/plain;charset=utf-8",
+            "text/plain",
+        ];
+
+        for target in &targets {
+            let atom = Atom::intern(target);
+            if let Some(selection) = clipboard.wait_for_contents(&atom) {
+                return std::str::from_utf8(&selection.get_data())
+                    .ok()
+                    .map(|text| text.to_string());
```

Comment:
Probably easier to just use the method on `String` directly, like:

```suggestion
                return String::from_utf8(&selection.get_data()).ok();
```

Reply:
I believe that I've mentioned before that I love the Rust standard library

Reply:
Ah ha, and because `String::from_utf8` can take ownership of the `Vec<u8>` returned by `.get_data()` we save a clone :)

Reply:
oh, right, so my suggestion isn't quite right, we don't want that borrow in there.